robottelo/ui/repository.py

```diff
@@ -90,6 +90,8 @@ def search(self, element_name):
         """
         self.navigate_to_entity()
         self.assign_value(locators['repo.search'], element_name)
+        prefix = 'kt_' if self.is_katello else ''
```

Comment:
these statements make sense only for base class. Here we already know that `is_katello = True`